mod/admin.php

```diff
@@ -282,14 +282,14 @@ function admin_page_federation(&$a) {
 	foreach ($platforms as $p) {
 		// get a total count for the platform, the name and version of the
 		// highest version and the protocol tpe
-		$c = q('SELECT COUNT(*) AS `total`, `platform`, `network`, `version` FROM `gserver`
+		$c = qu('SELECT COUNT(*) AS `total`, `platform`, `network`, `version` FROM `gserver`
 				WHERE `platform` LIKE "%s" AND `last_contact` > `last_failure` AND `version` != ""
```

Comment:
My regular strategy is to replace existing functionalities with completely rewritten codes in completely new files. These new files then should be checked with much care. @rabuzarus did so in the past with the documentation of the new protocol functions (and he really was a PITA - but a needed one).